src/site/_data/postTags.js

```diff
@@ -189,10 +189,17 @@ const tags = {
 
 const postTags = {};
 
-Object.keys(tags).forEach((key) => {
-  postTags[key] = {
-    ...tags[key],
-    href: `/tags/${key}/`,
+Object.keys(tags).forEach(($key) => {
+  const tag = tags[$key];
+  const description = tag.description
+    ? tag.description
+    : `Our latest news, updates, and stories about ${tag.title}.`;
```

Comment:
Random thought: strings like this are going to be painful when we start doing i18n.